### DISCUSSION TITLE
What Input Shaping Freq are you finding for your Ender 3 v2?

### DISCUSSION THREAD

**Discussion by Ergonomicmike** [mriscoc/Ender3V2S1#814] [General]
I'm curious to know if I am finding what others are finding on a stock (Bowden, glass plate) Ender 3 v2.

I don't see any ringing on X up to 60 Hz. So I disabled Input Shaping for X in the firmware.
I found 38 Hz for Y.

**Comment by dfinberg** [comment]
35 or so on my Neo Max I think.  Still a modest amount of noise, was going to try and retighten everything and recalibrate over the weekend.  What accelerations are you running at after tuning?

**Reply by Ergonomicmike** [reply]
I set 3000 on both x and y for the test and left them there.

My parts cooling is my limitation, followed by flow rate with a stock .4 mm nozzle. (I have a .6 CHT which eliminates that limit.)

I installed a second fan (5010?) and a custom duct. But a benchy shows the classic melting on the bow if I print 150 mm/s.

I can print a good benchy in 57 minutes (PLA). That's down from 1 hr, 43 minuties. So I'm content.

**Reply by dfinberg** [reply]
I cleaned things up a bit, and I'm at 2k acceleration, Y35, X16.5 on a Neo Max. The Klipper notes suggest that X value is pretty suspicious, but who knows, it looks better on the tower with it there.  It's looking ok at 100mm/s, but I think that's the limit of the stock neo hotend. I'm seeing what appears to be some underextrusion when I did some experiments the other day with a fast printing xmas ornament divider. I guess I need to decide if I want to invest in an upgraded hotend.
